tests/test_make.py

```diff
@@ -1,61 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Ok, then I think this is part of what the CI should be doing once we re-integrate it.